pkg/debug/spec.sh

```diff
@@ -331,14 +335,10 @@ print_usb_devices() {
         then
             parentassigngrp=""
         fi
-        local product
-        product=$(cat "$devicepath/product" 2> /dev/null || true)
         local label
-        label=$(echo "$product" | tr -cd '[:alnum:]._-')
-        if [ "$label" = "" ]
-        then
-            label="USB${busAndPort}"
-        fi
+        idVendor=$(cat "${devicepath}/idVendor")
+        idProduct=$(cat "${devicepath}/idProduct")
+        label="USB-${idVendor}:${idProduct}-${busAndPort}"
```

Comment:
Maybe for modems we could use label prefix `WWAN` and for ethernet NICs `ETH`?
Just not to confuse users as this will be used for network adapter name.
Also, currently the logical label is something like: `USB-1199:9091-3-9`. Should we use the same separator for device/product ID as for USB address? (i.e. to decide between `:` and `-`).